PVTPackage/source/MultiphaseSystem/MultiphaseSystemProperties/BlackOilFlashMultiphaseSystemProperties.cpp

```diff
@@ -28,6 +30,10 @@ BlackOilFlashMultiphaseSystemProperties::BlackOilFlashMultiphaseSystemProperties
     m_lnFugacity.insert( { pt, pvt::VectorPropertyAndDerivatives< double >( nComponents, nComponents ) } );
   }
 
+  m_lnFugacity.at( pvt::PHASE_TYPE::OIL ).value = { std::log( 1. ), std::log( 1. ), std::log( 1. ) };
+  m_lnFugacity.at( pvt::PHASE_TYPE::GAS ).value = { std::log( 1. ), std::log( 1. ), std::log( 1. ) };
+  m_lnFugacity.at( pvt::PHASE_TYPE::LIQUID_WATER_RICH ).value = { std::log( 1. ), std::log( 1. ), std::log( 1. ) };
+  
```

Comment:
Hmm, I realize that importing `cmath` just for computing zeros is a bit overkill.
Just `{0., 0., 0.}`?

Plus I think that you should put this in a `for( pvt::PHASE_TYPE pt: getPhases() )` loop so you are dealing with cases where you do not have three phases (is it possible?).

Even more: aren't the default values already `0`, in which case you do not need to do anything.

Reply:
Done, I removed this, it is already set to zero at initialization.